cuda/matrix/ell_kernels.cu

```diff
@@ -51,11 +55,16 @@ namespace ell {
 
 constexpr int default_block_size = 512;
 
+// TODO: multiple and ratio are parameters should be tuned
+constexpr int multiple = 4;
+constexpr double ratio = 1e-2;
```

Comment:
A comment what these are would be nice, or a more specific name (ratio of what?, multiple of what?).

Reply:
Minor: change the comment style to doxygen style (`/** <comment> */`) and move the comment above each variable.

Reply:
done